find_files.ps1

```diff
@@ -65,7 +65,7 @@ $HAS_SELECTION=VGet "env:HAS_SELECTION" 0
 $SELECTION_FILE=VGet "env:SELECTION_FILE" ""
 $QUERY=""
 if ($HAS_SELECTION -eq 1 -and "$SELECTION_FILE".Length -gt 0) {
-    $QUERY=Get-Content "$SELECTION_FILE" -Raw
+    $QUERY="`"$(Get-Content "$SELECTION_FILE" -Raw)`""
```

Comment:
Thanks for the PR. Did you test this? It looks to me like the quotes are applied incorrectly.

Reply:
I tried to fix the things we could fix. I sadly did not yet find a solution to everything, but at least the issue should be resolved